escoba/src/plugins/bootstrap-vue.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
+
+import BootstrapVue from 'bootstrap-vue'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+Vue.use(BootstrapVue)
```

Comment:
this line is already in main.js - seems like probably only needed once?

Reply:
i'm unclear about bootstrap actually.  it is imported as it's own dependency but then i used vue cli "vue add bootstrap-vue" to add a plugin.  this is because bootstrap requires jquery and jQuery and jquery.js, at least at work, and it a weird thing that may have to do with the repo we use there.